contents/blog/recommended-reading.md

```diff
@@ -0,0 +1,145 @@
+---
+date: 2023-08-16
+title: "PostHog's recommended reading for startup teams"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - joe-martin
+featuredImage: ../images/blog/super-hog-pink.png
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Guides
+---
+
+The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend. 
+
```

Comment:
```suggestion

There are thousands of books which will tell you how to build a product and launch a new startup, but it can be hard to know which are actually worth reading and which are full of generic advice (or written by AI). If you’re looking for inspiration, where should you turn?

The PostHog team includes a number of voracious readers — we even have [our own book club](/handbook/people/training)! — so here’s a collection of the books our teams recommend.

If you're looking for more startup advice or support, don't forget to check out [PostHog for Startups](/startups), our program for helping early-stage teams get to product market fit. Our startup program includes $50,000 in PostHog credit, exclusive events and promotional opportunities, and is available to any company less than 2 years old and with less than $5M in funding. 
```